sql/init.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+psql -U postgres -c "CREATE DATABASE leaderboard"
+psql -U postgres -d leaderboard -f schema.sql
+psql -U postgres -d leaderboard -f testdata.sql
```

Comment:
I think we may want to support this with and without testdata.sql. Seemed like overkill for this sample though.